.circleci/config.yml

```diff
@@ -17,8 +17,12 @@ commands:
     description: "Ensure built assets are up to date"
     steps:
       - checkout
-      - run: npm ci
-      - run: npm run build
+      - setup_remote_docker
+      # - run: npm ci
+      - run: docker build -t sphinx_rtd_theme:latest .
+      - run: docker run --cidfile=.container_id --mount type=bind,source="${CIRCLE_WORKING_DIRECTORY}/src",target=/project/src,readonly sphinx_rtd_theme:latest
+      - run: docker cp "$(cat .container_id):/project/sphinx_rtd_theme" .
+      - run: docker cp "$(cat .container_id):/project/package-lock.json" .
```

Comment:
Noted already, but the node orb should be used here instead of Docker.

Reply:
This is now replaced with a CircleCI env that builds our Docker image like we specify and it doesn't take a lot of build time :+1: